posts/saving-and-analyzing-trending-topics-on-twitter-using-aws-athena-lambda-and-cdk.md

```diff
@@ -0,0 +1,198 @@
+---
+authors:
+- Theo Lebrun
+tags:
+- Twitter
+- AWS
+- AWS Lambda
+- AWS Athena
+- AWS CDK
+date: 2020-08-03T14:50:55.000Z
+title: "Saving and analyzing trending topics on Twitter using AWS Athena, Lambda and CDK"
+image:
+---
+
+With more than 300 million active users, Twitter is still the best platform to provide efficient and accurate real time news. The SMS of the Internet is also one of the best social network and is particularly used a lot by all kind of developers which makes it a very good source of tech information.
+
+With #Hashtags, Twitter found a way to let the users group posts together and follow specific topics in order to get immediate updates. My favorite tool to give you the best Twitter experience is [TweetDeck](https://tweetdeck.twitter.com/), you can define your own custom dashboard in order to fulfill all your needs. My favorite "column" is the `Trending` one and my guilty pleasure is to check it multiple times a day to always be aware of what is happening (even if sometimes I am quite surprised of what is currently trending...).
+
+The goal of this blog is to explain how cheap and easy it is to save and analyze what is trending on Twitter using services provided by AWS. There are a lot of neat use cases that can be done once you have this architecture up and running. One for example would be to notify you if a specific topic is trending in order to not miss important news.
+
+# AWS Services and Architecture
+
+The beauty of AWS is that it provides us all the services that we need in order to build our Twitter analytics platform. That saves us a lot of time and headaches since we can focus on the fun parts of this project instead of less fun parts like hosting and configuration.
```

Comment:
That saves us time and headaches of hosting and configuration, and instead allows us to focus on the fun parts of this project.